src/components/organisms/AssetActions/index.tsx

```diff
@@ -2,28 +2,52 @@ import React, { ReactElement, useState, useEffect } from 'react'
 import styles from './index.module.css'
 import Compute from './Compute'
 import Consume from './Consume'
-import { Logger } from '@oceanprotocol/lib'
+import { Logger, File as FileMetadata, DID } from '@oceanprotocol/lib'
 import Tabs from '../../atoms/Tabs'
 import compareAsBN from '../../../utils/compareAsBN'
 import Pool from './Pool'
 import Trade from './Trade'
 import { useAsset } from '../../../providers/Asset'
 import { useOcean } from '../../../providers/Ocean'
 import { useWeb3 } from '../../../providers/Web3'
+import { getFileInfo } from '../../../utils/provider'
+import axios from 'axios'
 
 export default function AssetActions(): ReactElement {
   const { accountId } = useWeb3()
-  const { ocean, balance, account } = useOcean()
-  const { price, ddo, metadata } = useAsset()
+  const { config, ocean, balance, account } = useOcean()
+  const { price, ddo } = useAsset()
 
   const [isBalanceSufficient, setIsBalanceSufficient] = useState<boolean>()
   const [dtBalance, setDtBalance] = useState<string>()
+  const [fileMetadata, setFileMetadata] = useState<FileMetadata>()
```

Comment:
Looks like there should be a default value in here otherwise due to this `if (!file) return null`  check from the `File` component the component will not show the content. That's why you don't get the component. 
Might be set to an empty object or create an object with the data from the `ddo` until you get the data from provider

Reply:
Thought that the problem was elsewhere. Thanks, I'll fix it.